corehq/apps/data_cleaning/views/tables.py

```diff
@@ -127,8 +132,18 @@ def select_all(self, request, *args, **kwargs):
 
     @hq_hx_action("post")
     def apply_all_changes(self, request, *args, **kwargs):
-        # todo: this is a placeholder
-        return self.get(request, *args, **kwargs)
+        self.session.prepare_session_for_commit()
+        commit_data_cleaning.apply_async(
+            args=(self.session.session_id,),
```

Comment:
Do we have a standard for trailing commas in single-line sets of arguments? I feel like I most often see them excluded, unless the args are long enough to span multiple lines.

Reply:
it's best practice to leave the trailing comma in tuples